src/modal/component/accordion/Accordion.tsx

```diff
@@ -0,0 +1,36 @@
+import React, {useState} from "react";
+
+import AccordionItem from "./item/AccordionItem";
+import {AccordionData} from "./util/accordionTypes";
+
+interface AccordionProps {
+  accordionData: AccordionData[];
+}
+
+function Accordion({accordionData}: AccordionProps) {
+  const [clicked, setClicked] = useState("0");
```

Comment:
- Let's use a more descriptive name
- Do we need to string casting here?

```suggestion
  const [activeItem, setActiveItem] = useState(0);
```